addons/test_mail/tests/test_mail_security.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.test_mail.tests.common import TestMailCommon
+from odoo.exceptions import AccessError
+
+class TestSubtypeAccess(TestMailCommon):
+
+    def test_subtype_access(self):
+        """
+        The function aims to formally verify the access restrictions on mail.message.subtype for
+        normal and admin users. It ensures that normal users are unable to modify it,
+        while admin users possess the necessary privileges to modify it successfully.
+        """
+
+        test_subtype2 = self.env['mail.message.subtype'].create({
+            'name': 'Test',
+            'description': 'only description'
+        })
+
+        only_user_group = self.env['res.groups'].create({
```

Comment:
Why defining a new group ? I think checking wiht base.group_user is sufficient, unless you have something else in mind ?